vispy/color/colormap.py

```diff
@@ -3,12 +3,14 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 
 from __future__ import division  # just to be safe...
+import inspect
 
 import numpy as np
 
 from .color_array import ColorArray
 from ..ext.six import string_types
 from ..ext.cubehelix import cubehelix
+from ..ext import husl
```

Comment:
looks like you only ever use `husl.husl_to_rgb` so might be worth just importing that directly as `from ..ext.husl import husl_to_rgb`